### DISCUSSION TITLE
Save (bookmark) place in GitHub mobile

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43959] [Mobile] [answered]
I'm reading a discussion or an issue or a pull request and I need to pull up information from a repository/issue/thing.

I'd love to be able to bookmark my spot, and then go to the main GitHub Mobile view, browse to get the information I need, copy/screenshot it, and then return to provide that information.

Here's one example of this:
https://github.com/community/community/discussions/34#discussioncomment-4656787

It happened to me multiple times while trying to triage [issues in my repository](https://github.com/check-spelling/check-spelling/issues).

**Comment by HenSquared** [comment, marked as answer]
Being able to bookmark a place would be helpful! I've heard similar feedback previously, so I've opened an internal issue to discuss this among the team. Thanks!

**Reply by Madhukarmolugri** [reply]
ok

**Comment by ghost** [comment]
> I'm reading a discussion or an issue or a pull request and I need to pull up information from a repository/issue/thing.
> 
> I'd love to be able to bookmark my spot, and then go to the main GitHub Mobile view, browse to get the information I need, copy/screenshot it, and then return to provide that information.
> 
> Here's one example of this:
> https://github.com/community/community/discussions/34#discussioncomment-4656787
> 
> It happened to me multiple times while trying to triage [issues in my repository](https://github.com/check-spelling/check-spelling/issues).
